packages/dates/src/tests/utilities.test.ts

```diff
@@ -1,9 +1,21 @@
 import {formatDate} from '../utilities/formatDate';
 
 describe('formatDate()', () => {
-  it('formats dates into YYYY-MM-DD', () => {
+  it('formats dates as expected for various locales', () => {
```

Comment:
We should be able to leverage jest `it.each` for things like this. 

```
 it.each([
    ['en-US', '12/31/2017'],
    ['en-GB', '31/12/2017'],
    ['de-DE', '31.12.2017'],
 ])(`formats using the locale $s and date $s`, (locale, date) => {
    const baseDate = new Date(2018, 0, 0);
    const formatedDate = `[${locale}] ${formatDate(baseDate, locale)}`;
    expect(formatedDate).toBe(`[${locale}] ${date}`);
  });
```

Reply:
Personally I prefer tests that aren't generated, it's way harder to find the test in the codebase when the name doesn't actually exist anywhere.

Reply:
I'm happy to follow direction here, but need to know whose to follow.  ;)

For the time being, I've reworked these tests to use `it.each()`.  I think that that should not be too hard to track down, because we can always `grep` for the text given to `describe()`.
(Also, the output of `dev test` shows the file and line number where the failure happened, which makes it easy to find the offending test.)